terms/boundary_constraints.md

```diff
@@ -0,0 +1,8 @@
+---
+comments: true
+---
+# Boundary Constraints
+
+Boundary constraints specify that at any designated point within a computation, a particular register must hold a specified value.
```

Comment:
Boundary constraints: at the start or at the end of the computation an indicated register has a given value. ?/